projects/dsp-ui/src/lib/viewer/services/value-type.service.ts

```diff
@@ -0,0 +1,64 @@
+import { Injectable } from '@angular/core';
+import {
+    Constants,
+    ReadTextValueAsHtml,
+    ReadTextValueAsString,
+    ReadTextValueAsXml,
+    ReadValue
+} from '@dasch-swiss/dsp-js';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class ValueTypeService {
```

Comment:
Good that you made this a service!

Reply:
Couldn't you use this service also in `add-value.component.ts`? Or is the problem the text value the reason you can't?

Reply:
I don't really understand the second question but `getValueTypeOrClass` takes a `ReadValue` as input which we don't have in `add-value.component.ts`

Reply:
Ok, understood. 

My second question was related to https://github.com/dasch-swiss/dsp-ui-lib/pull/109#discussion_r452263583